### PR TITLE
added a drush version 6.5 formula

### PR DESCRIPTION
**Pull request by jhuebsch**
A copy of the drush formula in head when it was still drush 6.5. Changes from that include: Added desc, updated hash to sha256 and made keg_only.

### CODE REVIEW

**Comment by jhuebsch** [comment]
Strange that brew audit failed with a desc being too long. Running brew audit locally I do not get that error.

File: drush6.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Drush6 < Formula
+  desc "A command-line shell and scripting interface for Drupal"
+  homepage "https://github.com/drush-ops/drush"
+  url "https://github.com/drush-ops/drush/archive/6.5.0.tar.gz"
+  sha256 "0e6bfb8a4bd75846dbca25f6d3e3130a706546f6a62edadce2eb00099750e2a7"
+  keg_only "Conflicts with drush in main repository."
+
+  head do
```
Comment (DomT4) [inline drush6.rb]: We don't really do `head` builds in Versions.

File: drush6.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    url "https://github.com/drush-ops/drush.git"
+
+    depends_on "Homebrew/php/composer" => :build
+  end
+
+  def install
+    system "composer", "install" if build.head?
+
+    prefix.install_metafiles
```
Comment (DomT4) [inline drush6.rb]: This happens automatically now. Can be removed.

File: drush6.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    File.delete "drush.bat"
+    libexec.install Dir["*"]
+    (bin+"drush").write <<-EOS.undent
+      #!/bin/sh
+      export ETC_PREFIX=${ETC_PREFIX:=#{HOMEBREW_PREFIX}}
```
Comment (DomT4) [inline drush6.rb]: Would this still be true under `keg_only`?

File: drush6.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      export SHARE_PREFIX=${SHARE_PREFIX:=#{HOMEBREW_PREFIX}}
+      exec "#{libexec}/drush" "$@"
+    EOS
+    bash_completion.install libexec/"drush.complete.sh" => "drush"
+  end
+
+  test do
+    system "drush", "version"
```
Comment (DomT4) [inline drush6.rb]: `bin/"drush"`

**Comment by DomT4** [comment]
> Strange that brew audit failed with a desc being too long. Running brew audit locally I do not get that error.

`brew audit --strict`

**Comment by DomT4** [comment]
Ping?

**Comment by jhuebsch** [comment]
I'd like to answer your questions but really I am unable to. The formula requirements and format have changed significantly since last time I added a drush version. I mistakenly thought I could just copy the formula from when drush was 6.5 that does not look to be the case.

**Pull request closed without merging**